ballerina-tests/src/test/resources/logging.properties

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+####################### HANDLERS #######################
+# Configurations for console logging
+java.util.logging.ConsoleHandler.level=ALL
+java.util.logging.ConsoleHandler.formatter=org.ballerinalang.logging.formatters.BallerinaLogFormatter
+org.ballerinalang.logging.formatters.BallerinaLogFormatter.format=%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS,%1$tL %2$s [%3$s] - %4$s %n
+
+org.ballerinalang.stdlib.common.TestLogHandler.level=INFO
+org.ballerinalang.stdlib.common.TestLogHandler.formatter=org.ballerinalang.logging.formatters.DefaultLogFormatter
+org.ballerinalang.logging.formatters.DefaultLogFormatter.format=[%1$tY-%1$tm-%1$td %1$tH:%1$tM:%1$tS,%1$tL] %2$s [%3$s] - %4$s %5$s %n
+
+####################### LOGGERS #######################
+# Ballerina user level root logger
+ballerina.handlers=java.util.logging.ConsoleHandler
+ballerina.level=SEVERE
+ballerina.useParentHandlers=false
+
+# JUL root logger
+.handlers=org.ballerinalang.stdlib.common.TestLogHandler
+.level=SEVERE
+
+com.zaxxer.hikari.level=SEVERE
+org.quartz.level=SEVERE
```

Comment:
I think we don't need these lines, do we ?

Reply:
Removed in https://github.com/ballerina-platform/module-ballerina-observe/pull/1/commits/9073340c438168a8545c0490b49c6092332b6138